hooks/tk-houdini_actions.py

```diff
@@ -116,4 +137,46 @@ def _merge(self, path, sg_publish_data):
         hou.hipFile.merge(path,
                           node_pattern="*",
                           overwrite_on_conflict=False,
-                          ignore_load_warnings=False)       
+                          ignore_load_warnings=False) 
+
+
+    ##############################################################################################################
+    def _import(self, path, sg_publish_data):
+
+        import hou
+        app = self.parent
+
+        name = sg_publish_data.get("name", "alembic")
+        path = self.get_publish_path(sg_publish_data)
+
+        obj_context = hou.node("/obj")
```

Comment:
If the current context could contain a geo node, seems like we should create it there (right?).
That way if you are deep in a subnetwork you could import the cache there.